packages/cookie-theme-toggler/src/hooks.ts

```diff
@@ -0,0 +1,6 @@
+import { useContext } from 'react'
+import { ThemeToggleContext, ThemeContext } from './cookie-theme-provider'
+
+export const useThemeToggle = (): ThemeContext => {
```

Comment:
What is the benefit of such hook?

Reply:
it was not me, but via using approach like this we're making it impossible to use this provider in custom manner